plugins/lsp/language-servers/efmls-configs.nix

```diff
@@ -7,11 +7,15 @@
 }:
 let
   tools = import ../../../generated/efmls-configs.nix;
-  packagingStatus = import ./efmls-configs-pkgs.nix pkgs;
+  packagingStatus = import ./efmls-configs-pkgs.nix lib;
 
   # Filter packages that are not compatible with the current platform
   toolPkgs = lib.filterAttrs (
-    a: pkg: lib.meta.availableOn pkgs.stdenv.hostPlatform pkg
+    name: loc:
+    let
+      pkg = lib.attrByPath (lib.toList loc) null pkgs;
```

Comment:
If I add a warning to the `null` fallback here, I get to the bottom of the issue in https://github.com/nix-community/nixvim/pull/2160#discussion_r1743722663:
```suggestion
      loc' = lib.toList loc;
      pkg = lib.attrByPath loc' (lib.warn "${lib.concatStringsSep "." loc'} cannot be found in pkgs!" null) pkgs;
```

```
trace: evaluation warning: python3.autopep8 cannot be found in pkgs!
trace: evaluation warning: eslint cannot be found in pkgs!
trace: evaluation warning: python3.flake8 cannot be found in pkgs!
trace: evaluation warning: python3.mdformat cannot be found in pkgs!
trace: evaluation warning: python3.vulture cannot be found in pkgs!
trace: evaluation warning: python3.autopep8 cannot be found in pkgs!
trace: evaluation warning: eslint cannot be found in pkgs!
trace: evaluation warning: python3.flake8 cannot be found in pkgs!
trace: evaluation warning: python3.mdformat cannot be found in pkgs!
trace: evaluation warning: python3.vulture cannot be found in pkgs!
trace: evaluation warning: python3.autopep8 cannot be found in pkgs!
trace: evaluation warning: eslint cannot be found in pkgs!
trace: evaluation warning: python3.flake8 cannot be found in pkgs!
trace: evaluation warning: python3.mdformat cannot be found in pkgs!
trace: evaluation warning: python3.vulture cannot be found in pkgs!
```

Reply:
Fixed by allowing "scope" to be a nested attr name and moving `python3` packages to `python3.pks`.